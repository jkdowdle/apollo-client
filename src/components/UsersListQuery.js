import gql from 'graphql-tag'

export const UsersQuery = gql`
query UsersQuery {
  users {
    id
    firstName
    age
    createdAt
  }
}
`