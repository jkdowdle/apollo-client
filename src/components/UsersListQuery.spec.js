import { UsersQuery } from './UsersListQuery'

it('should be the correct query', () => {
  expect(UsersQuery).toMatchSnapshot()
})