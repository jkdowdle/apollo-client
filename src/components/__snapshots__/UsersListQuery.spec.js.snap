// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be the correct query 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "kind": "OperationDefinition",
      "name": Object {
        "kind": "Name",
        "value": "UsersQuery",
      },
      "operation": "query",
      "selectionSet": Object {
        "kind": "SelectionSet",
        "selections": Array [
          Object {
            "alias": null,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "users",
            },
            "selectionSet": Object {
              "kind": "SelectionSet",
              "selections": Array [
                Object {
                  "alias": null,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "id",
                  },
                  "selectionSet": null,
                },
                Object {
                  "alias": null,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "firstName",
                  },
                  "selectionSet": null,
                },
                Object {
                  "alias": null,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "age",
                  },
                  "selectionSet": null,
                },
                Object {
                  "alias": null,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "createdAt",
                  },
                  "selectionSet": null,
                },
              ],
            },
          },
        ],
      },
      "variableDefinitions": Array [],
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 79,
    "start": 0,
  },
}
`;
